tests/behat/features/capabilities/event/filter-community-events.feature

```diff
@@ -0,0 +1,50 @@
+@api @javascript @event @stability @stability-1 @filter-community-events
+Feature: Filter community events
+  Benefit: Ability to filter events on All events page
+  Role: As a Verified
+  Goal/desire: I want to filter events that have all day checkbox activated
+
+  Scenario: Successfully use filters for ongoing and upcoming events with today's end date
+    Given users:
+      | name               | mail             | status | timezone | roles    |
+      | regular_user       | some@example.com | 1      | UTC      | verified |
+    Given events with non-anonymous author:
+      | title                     | body        | field_content_visibility | field_event_date    | field_event_date_end | field_event_all_day	 | langcode |
+      | Upcoming or Ongoing Event | lorem ipsum | public                   | today               | today                | 1                    | en       |
+    When I am logged in as "regular_user"
+    Given I am on the event overview
+    And I should see "Filter" in the "Sidebar second"
+    And I should see "Ongoing and upcoming events"
+    And I click radio button "Ongoing and upcoming events"
+    And I press "Filter"
+    And I should see "Upcoming or Ongoing Event" in the "Main content"
+
+  Scenario: Successfully use filters for ongoing and upcoming events
+    Given users:
+      | name               | mail             | status | timezone | roles    |
+      | regular_user       | some@example.com | 1      | UTC      | verified |
+    Given events with non-anonymous author:
+      | title                     | body        | field_content_visibility | field_event_date    | field_event_date_end | field_event_all_day	 | langcode |
+      | Upcoming or Ongoing Event | lorem ipsum | public                   | 2049-01-01          | 2049-01-01           | 1                    | en       |
+    When I am logged in as "regular_user"
+    Given I am on the event overview
+    And I should see "Filter" in the "Sidebar second"
+    And I should see "Ongoing and upcoming events"
+    And I click radio button "Ongoing and upcoming events"
+    And I press "Filter"
+    And I should see "Upcoming or Ongoing Event" in the "Main content"
```

Comment:
Similar suggestions as for the previous scenario, but put in a single block.

This test will also start failing in 2049. There's a good chance we won't be working here anymore by then, but a poor soul might still be using this test, coming back from their New Years holiday 😛  so this might be a good place to use a relative date (like `+ 1 week`), I've not included it in the suggestion but you can try it out yourself.

```suggestion
  Scenario: Successfully use filters for ongoing and upcoming events
    Given users:
      | name               | mail             | status | timezone | roles    |
      | regular_user       | some@example.com | 1      | UTC      | verified |
    And events with non-anonymous author:
      | title                     | body        | field_content_visibility | field_event_date    | field_event_date_end | field_event_all_day	 | langcode |
      | Upcoming or Ongoing Event | lorem ipsum | public                   | 2049-01-01          | 2049-01-01           | 1                    | en       |
    And I am logged in as "regular_user"
    
    When I am on the event overview
    And I click radio button "Ongoing and upcoming events"
    And I press "Filter"
    
    Then I should see "Upcoming or Ongoing Event" in the "Main content"
```